libsolutil/FixedHash.h

```diff
@@ -114,7 +122,16 @@ class FixedHash
 	bool operator==(FixedHash const& _c) const { return m_data == _c.m_data; }
 	bool operator!=(FixedHash const& _c) const { return m_data != _c.m_data; }
 	/// Required to sort objects of this type or use them as map keys.
-	bool operator<(FixedHash const& _c) const { for (unsigned i = 0; i < N; ++i) if (m_data[i] < _c.m_data[i]) return true; else if (m_data[i] > _c.m_data[i]) return false; return false; }
+	bool operator<(FixedHash const& _c) const {
+		for (unsigned i = 0; i < N; ++i)
+		{
+			if (m_data[i] < _c.m_data[i])
+				return true;
+			else if (m_data[i] > _c.m_data[i])
+				return false;
+		}
+		return false;
+	}
```

Comment:
```suggestion
	bool operator<(FixedHash const& _c) const { return m_data < _c.m_data; }
```

Actually, I'm not sure, if it wouldn't be best to just implement the whole class as:
```
template<size_t N> using FixedHash = std::array_t<uint8_t, N>;
```
and adjust everything else around that...

Reply:
Yep, that was my goal to reduce this class. I have other branches to do that, but first wanted to create tests to compare those changes against.